pkg/reconciler/contour/config/contour.go

```diff
@@ -39,6 +40,7 @@ const (
 	defaultTLSSecretConfigKey = "default-tls-secret"
 	timeoutPolicyIdleKey      = "timeout-policy-idle"
 	timeoutPolicyResponseKey  = "timeout-policy-response"
+	corsPolicy                = "corsPolicy"
```

Comment:
We use `snake-case` in config maps

```suggestion
	corsPolicy                = "cors-policy"
```

Reply:
addressed